includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -146,6 +149,53 @@ private function get_quiz_questions( WP_Post $quiz ) : array {
 			return [];
 		}
 
+		$quiz_questions     = [];
+		$multiple_questions = [];
+		foreach ( $questions as $question ) {
+			if ( 'multiple_question' === $question->post_type ) {
+				$multiple_questions[] = $question;
+			} else {
+				$quiz_questions[] = $this->get_question( $question );
+			}
+		}
+
+		foreach ( $multiple_questions as $multiple_question ) {
+			$quiz_questions = array_merge( $quiz_questions, $this->get_questions_from_category( $multiple_question, wp_list_pluck( $quiz_questions, 'id' ) ) );
+		}
+
+		return $quiz_questions;
+	}
+
+	/**
+	 * This method retrieves questions as they are defined by a 'multiple_question' post type.
+	 *
+	 * @param WP_Post $multiple_question  The multiple question.
+	 * @param array   $excluded_questions An array of question ids to exclude.
+	 *
+	 * @return array
+	 */
+	private function get_questions_from_category( WP_Post $multiple_question, array $excluded_questions ) : array {
+		$category = (int) get_post_meta( $multiple_question->ID, 'category', true );
+		$number   = (int) get_post_meta( $multiple_question->ID, 'number', true );
+
+		$args = [
+			'post_type'        => 'question',
+			'posts_per_page'   => $number,
+			'orderby'          => 'rand',
```

Comment:
I don't think we want these randomized, it should be consistent. The current implementation for the frontend also just orders by title and always takes the first X questions (if the randomize setting is not selected).

Reply:
Yep I was thinking that this can cause issues too. Fixed in a6614ff608c088dd5ecf5422aa03dbfd6757aaf9.

It's just that when picking a subset of the questions, it doesn't make much sense to always pick them according to an ordering which isn't obvious to the user. But we shouldn't change it here.